db/crud.go

```diff
@@ -373,6 +373,53 @@ func (db *DatabaseCollectionWithUser) getRev(ctx context.Context, docid, revid s
 	return revision, nil
 }
 
+func (db *DatabaseCollectionWithUser) GetCV(ctx context.Context, docid string, cv *Version, includeBody bool) (revision DocumentRevision, err error) {
```

Comment:
There's a lot of code cloned from GetRev in here.  Did it seem feasible to use a shared function for the common code (i.e. everything after revision retrieval)?

Reply:
Should be done with the new `documentRevisionForRequest` function.